README.md

```diff
@@ -151,6 +151,11 @@ Congratulations, if the last command worked, you now have kubernetes running in
      ```
      This is necessary, because the default unit definition also defines a host address, so that it cannot be overridden by the configuration file.
 
+     You might need to restart docker in the lxc container. Run
```

Comment:
`might need to` -> `should`. I think this step was simply missing and everybody needs to do this, right?